pages/api/constants.ts

```diff
@@ -0,0 +1,11 @@
+export const SAML_GENERATION_URL = process.env.NEXT_PUBLIC_SAML_GENERATION_URL;
+export const CLIENT_ID = process.env.NEXT_PUBLIC_COGNITO_CLIENT_ID!;
```

Comment:
Why the `!` but the fallback in the one below?

Reply:
This was a fallback, and then Matt requested I change it to a `!`... and I didn't normalize the file with one choice.

I should normalize the file with one choice and I think that should probably be the `!`, but I have never built an Amplify app myself and I was not sure if the error message would be clear enough when env variables are empty.

I will change both to `!` but I do want to do more research/testing about this sometime soon.